tests/test_aggregation.py

```diff
@@ -176,6 +176,7 @@ async def test_agg_query(self):
             count_list[k] += 1
           k+=1
         logger.info(f'Time of runs: {i+1}, Successful count: {count_list}')
+        assert any(count_list)

```

Comment:
'any()' will raise an error if all elements in the list are 0, this is not what we want. Better to only allow one fail in total: 'sum(count_list) >= len(count_list) * num_runs - 1'.